packages/framer-motion/src/frameloop/data.ts

```diff
@@ -0,0 +1,4 @@
+export const frameData = {
```

Comment:
Is this just a public const?

Reply:
Yeah - the props are mutable so we don't write a new object every frame, it's actually always been this way and I think its mutable nature was masked slightly by the `getFrameData()` api